README.md

```diff
@@ -1,14 +1,16 @@
+# bcache
 These are the userspace tools required for bcache.
 
-Bcache is a patch for the Linux kernel to use SSDs to cache other block
+`bcache` is a patch for the Linux kernel (developed in the kernel source tree) 
```

Comment:
That's not my text. Feel free to fix the original, then I'll rebase this PR on it.